community/contacts.md

```diff
@@ -0,0 +1,21 @@
+# Contact Information
+
+The *OpenSSL Software Foundation* (OSF) represents the OpenSSL project in
+most legal capacities including contributor license agreements, managing
+donations, and so on. It is a Delaware (US) non-profit corporation with its
+own [bylaws](/policies/osf-bylaws.pdf).
+
+*OpenSSL Software Services* (OSS) also represents the OpenSSL project, for
+[Support Contracts](/support/contracts.html), and as the Vendor of Record
+for NIST Cryptographic Module
+[\#1747](https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/1747)
+(This is an open-source validation of [FIPS-140](/docs/fips.html) based on
+OpenSSL). It is a Delaware (US) corporation with its own
+[bylaws](/policies/oss-bylaws.pdf).
+
+The best way to contact OSF or OSS is by sending an email to
+<osf-contact@openssl.org>. For postal contact, use the following:
+
+> 40 E Main St, Suite 744\
+> Newark DE 19711\
+> USA\
```

Comment:
This \\ appears in the rendered output.  I suspect none are required.

Reply:
The last one isn't needed, that's right.